src/os/darwin/darwin_sigar.c

```diff
@@ -29,6 +29,7 @@
 #include <nfs/rpcv2.h>
 #include <nfs/nfsproto.h>
 #include <nfs/nfs.h>
+#ifndef NFS_NPROCS
 #if (__MAC_OS_X_VERSION_MIN_REQUIRED >= 101406)
 struct nfsstats {
```

Comment:
This structure has existed in the MacOS full toolchain since at least before 10.1.5, although when testing the build in 10.14 with command line tools only this did not fail. I suspect this guard is not going to be right for all platforms and definition should be introduced only when the target toolchain compiled against is 12.x or greater.

```suggestion
#include <nfs/nfs.h>
#if (__MAC_OS_X_VERSION_MIN_REQUIRED >= 120001)
struct nfsstats {
```